share/harvesters/org_swbiodiversity.py

```diff
@@ -0,0 +1,127 @@
+import itertools
+import re
+import logging
+
+from bs4 import BeautifulSoup, Comment
+
+from share.harvest import BaseHarvester
+
+
+logger = logging.getLogger(__name__)
+
+
+class SWHarvester(BaseHarvester):
+    """
+
+    """
+    VERSION = 1
+
+    def do_harvest(self, start_date, end_date):
+        end_date = end_date.date()
+        start_date = start_date.date()
+        logger.info('Harvesting swbiodiversity %s - %s', start_date, end_date)
+        return self.fetch_records()
+        # get ExPORTER page html and rows storing records
+
+    def fetch_records(self):
+        html = self.requests.get(self.kwargs['list_url'])
+        html.raise_for_status()
+        soup = BeautifulSoup(html.content, 'lxml')
+        records = soup.find_all('a')
+        record_list = []
+        for record in records:
+            record_content = re.findall('collid=(\d+)', record.get('href'))
+            if record_content and record_content[0] not in record_list:
+                record_list.append(record_content[0])
+        total = len(record_list)
+
+        logging.info('Found %d results from swbiodiversity', total)
+        count = 0
+        while count < total:
+
+            logger.info('On collection %d of %d (%d%%)', count, total, (count / total) * 100)
+            data = {}
+            identifier = record_list[count]
+
+            data['identifier'] = identifier
+            html = self.requests.get(self.kwargs['list_url'] + '?collid=' + identifier)
+            html.raise_for_status()
+            soup = BeautifulSoup(html.content, 'lxml')
+
+            # Peel out script tags and css things to minimize size of HTML
+            for el in itertools.chain(
+                    soup('img'),
+                    soup('link', rel=('stylesheet', 'dns-prefetch')),
+                    soup('link', {'type': re.compile('.')}),
+                    soup('noscript'),
+                    soup('script'),
+                    soup(string=lambda x: isinstance(x, Comment)),
+            ):
+                el.extract()
+
+            record = soup.find(id='innertext')
+            title = self.process_text(record.h1)
```

Comment:
If you still prefer to give this data to the transformer, you can massage the data before-hand by overriding `transform`